egui/src/widgets/plot/items/mod.rs

```diff
@@ -1365,7 +1371,13 @@ impl PlotItem for BarChart {
         find_closest_rect(&self.bars, point, transform)
     }
 
-    fn on_hover(&self, elem: ClosestElem, shapes: &mut Vec<Shape>, plot: &PlotConfig<'_>) {
+    fn on_hover(
+        &self,
+        elem: ClosestElem,
+        shapes: &mut Vec<Shape>,
+        plot: &PlotConfig<'_>,
+        _: &CustomLabelFunc,
```

Comment:
Currently implemented only for the line chart. If this is the direction that we are good to go, happy to continue implementing for BarChart and BoxPlot.